client/student/app/lecture/comment.panel.directive.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+    'use strict';
+    
+    angular
+        .module('student')
+        .directive('commentPanel', commentPanel);
+    
+    function commentPanel() {
+        var directive = {
+            restrict: 'E',
+            templateUrl: 'app/lecture/comment.panel.html',
+            link: link,
+            controller: CommentPanelController,
+            controllerAs: 'vm',
+            bindToController: true
+        };
+        
+        return directive;
+        
+        function link(scope, element, attrs, vm) {
+            console.log('Ready (Archive Link)');
+        }
+    }
+    
+    /* @ngInject */
+    function CommentPanelController(lectureFactory) {
+        console.log('Ready (Simple List Panel Controller)');
+        
+        getComments();
+        
+        var vm = this;
+        vm.list = [];
+        
+        function getComments() {
+            lectureFactory
+                .getComments(function(response) {
+                    console.log('Success', response);
+                    vm.list = response;
```

Comment:
The API specifies that the response is an object with a property called 'comments'. This property is an array with all the comment objects. So this line should be `vm.list = response.comments`.